src/forms/output/chartStringTransformers.js

```diff
@@ -2,32 +2,38 @@
  * @module */
 
 
-/** Recursively extract text from language-specific React
- *     objects. Creates one inline string of text.
- * Recursion is untested.
+/** Mutator. Set the thousands separator character for a Highcharts`
+ *     options.
  *
- * @params {object} translationObj React element containing
- *     children that are strings or other React elements.
+ * @params {object} translations The strings for the app's current
+ *     language.
+ * @params {object} Highcharts Where to set the options.
  *
- * @returns {string}
+ * @returns {nothing}
```

Comment:
I *think* the convention for functions with no return value in JSDoc is the type `void`, but Googling it comes up with fairly disparate suggestions (e.g. Closure Compiler says don't include the tag at all if it returns nothing). So maybe there's no consensus.

Reply:
Hmm, good note. We should decide on our own convention and then document it. @todo discussion.